assets/pages/Container.vue

```diff
@@ -91,4 +91,8 @@ export default {
 .is-fullheight {
   min-height: 100vh;
 }
+
+.item-message {
```

Comment:
There is already a class `.text`. Can't you use that? Since Vuejs has the scoped CSS, you don't need to worry about class collisions.

Reply:
Changed it, thanks for explaining. I'm new to vue.